src/core/plugins/spec/actions.js

```diff
@@ -481,9 +481,19 @@ export const executeRequest = (req) =>
             err.name = ""
             err.message = "**Failed to fetch.**  \n**Possible Reasons:** \n  - CORS \n  - Network Failure \n  - URL scheme must be \"http\" or \"https\" for CORS request."
           }
-          specActions.setResponse(req.pathName, req.method, {
-            error: true, err: serializeError(err)
-          })
+          if (err.response.data !== undefined && err.response.data !== null) {
+            const res = serializeError(err)
+            res.response.data = err.response.data
```

Comment:
This change technically goes against the nature of serializing errors. We're serializing the error, but then we attaching unserializable data to it. Let's think what else we can do here.

Reply:
As discussed, serialization should be part of redux reduction and not redux action emission.